lesson/outline/outline-template.md

```diff
@@ -1,104 +1,323 @@
-# Lesson Name
-
-Check out the [content standards](http://curriculum-documentation.codecademy.com/Content-Standards/outline/) for guidance on writing lesson outline.
+# Testing in Jest
 
 ## Lesson Information
 
 ### Resource(s)
 
-_Insert any resources you used while researching to help plan your lesson._
+- [jest official documentation](https://jestjs.io/)
+
 
 ### Learning Standards
-Check out the [content standards](http://curriculum-documentation.codecademy.com/Resources/outcomes-standards-objectives/#learning-standards) for guidance on writing learing standards.
 
-1. ...
-2. ...
-3. ...
+1. What is Jest
+2. Installing Jest
+3. Running Jest Test
+4. The `test()` function
+5. The `expect()` function and matchers
+6. False Positives in Jest with Asynchronous Code
+7. Testing Async Code - Async Callbacks
+8. Testing Async Code - Promises
+9. Mocking functions with `jest.fn()`
+10. Mocking modules with `jest.mock()`
+
 
 <hr>
 
-## Exercise # 1: _Insert exercise title here_
+## Exercise # 1: What is Jest
 
 ### Which course outcomes will be covered by this exercise?
 
-1. Learners will be able to...
+1. Learners will obtain a general understanding of testing and why it is important. 
+2. Learners will get brief exposure to the three main types of tests: 
+    - unit tests
+    - integration tests
+    - end-to-end tests. 
+3. Learners will understand the viability and efficiency of performing tests with jest. 
+
 
 ### Narrative Summary
 
-_Check out the [content standards](http://curriculum-documentation.codecademy.com/Content-Standards/narrative/) for guidance on writing narratives for exercises._
+1. In this lesson, we will explore the wonders of testing with jest and how it can expedite the project creation process. While there are many testing libraries out there, Jest provides a testing framework that focuses on simplicity, allowing us to do a lot, right out of the box without much setup.
```

Comment:
I think this is a great sentence for this exercise but would be better at the end rather than the beginning. Before talking about Jest as a testing library, first, define what testing is and what a testing library is.